pkg/agent/cniserver/server.go

```diff
@@ -760,10 +759,7 @@ func (s *CNIServer) reconcile() error {
 	klog.InfoS("Starting reconciliation for CNI server")
 	// For performance reasons, use ResourceVersion="0" in the ListOptions to ensure the request is served from
 	// the watch cache in kube-apiserver.
```

Comment:
The comment should be moved together to the new function.

Reply:
updated.